pyproject.toml

```diff
@@ -48,8 +48,8 @@ test = [
 
 docs = [
     "volatility3[dev]",
-    "sphinx>=8.0.0,<7",
```

Comment:
I've tested and it appears to builds with version 8 ok, so I'm not sure this needs to drop to 7, I think the higher version should bump to 9.